README.md

```diff
@@ -29,3 +29,52 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+Recall my implementation,
+```js
+function flip(array, r) {
+    var l = 0;
+    while(l < r){
+      var temp = array[r];
+      array[r] = array[l];
+      array[l] = temp;
+      l++;
+      r--;
+    }
+}
+
+// This will sort by first finding the maximum element in the not sorted part of the array then it will flip that element to the front and finally it will flip that to the very back
+function pancakeSort(array) {
+  for (var i = array.length; i > 1; i--){
+    var maxindex = panhelp(array,i);
+    if(maxindex != i - 1){
+      flip(array, maxindex);
+      flip(array, i - 1);
+    }
+  }
+    return array;
+}
+
+// This function will find the largest element in the array
+function panhelp(array, size){
+  var index = 0;
+
+  for (var i = 1; i < size; i++){
+    if (array[i] > array[index]) index = i;
+  }
+  return index;
+}
+```
+
+## Comparisons
+To find the max element in the unsorted array it needs to look through the entire unsorted array each iteration, this means $\Theta(n^2)$ comparisons.
+
+## Flips
+The way that the sorting works is that it will first finding the maximum element in the not sorted part of the array then it will flip that element to the front and finally it will flip that to the very back. So for every element it will flip it twice, $2n$ flips. Thus the asymptotic runtime is $\Theta(n)$. 
```

Comment:
Runtime or number of flips?

Reply:
My mistake. It should be number of flips.